usaspending_api/api_contracts/contracts/search/AdvancedSearch.md

```diff
@@ -319,6 +319,7 @@ List of table columns
 + `contract_pricing_type_codes`: `J` (optional, array[string])
 + `set_aside_type_codes`: `NONE` (optional, array[string])
 + `extent_competed_type_codes`: `A` (optional, array[string])
++ `tas_codes` (optional, array[TASCodeObject], fixed-type)
```

Comment:
These contracts need to include this new filter key and object

`usaspending-api/usaspending_api/api_contracts/contracts/search/SpendingByAwardCount.md` 
`usaspending-api/usaspending_api/api_contracts/contracts/search/SpendingByGeography.md`